docs/advanced_topics/images/feature_detection.rst

```diff
@@ -5,17 +5,22 @@ Feature Detection
 
 Wagtail has the ability to automatically detect faces and features inside your images and crop the images to those features.
 
-Feature detection uses OpenCV to detect faces/features in an image when the image is uploaded. The detected features stored internally as a focal point in the ``focal_point_{x, y, width, height}`` fields on the ``Image`` model. These fields are used by the ``fill`` image filter when an image is rendered in a template to crop the image.
+Feature detection uses third-party tools to detect faces/features in an image when the image is uploaded. The detected features stored internally as a focal point in the ``focal_point_{x, y, width, height}`` fields on the ``Image`` model. These fields are used by the ``fill`` image filter when an image is rendered in a template to crop the image.
```

Comment:
The detected features **are** stored internally

Reply:
Fixed by 5ea9e81